sources/core/Stride.Core/Diagnostics/ChromeTracingProfileWriter.cs

```diff
@@ -92,6 +97,7 @@ public void Stop()
                 writerTask?.Wait();
```

Comment:
Why does it need to wait after `Stop()` has been called. Can't it just signal a cancellation and return immediately?

Reply:
I think it would be good to wait here in order to observe any exception which may have been thrown